mario/core/AttrData.py

```diff
@@ -1390,180 +1397,238 @@ def to_pymrio(
+   
+    def add_sectors(
+        self,
+        io:str = 'inventories',
+        scenario:str = 'baseline',
+        inplace: bool = True,
+        ignore_warnings: bool = True,
+        notes=None,
+    ):        
+        """
+        Adds inventories to the database as new sectors/commodities/activities.
+        N.B. This method will erase all other scenarios different from the provided one and will return a new instance with scenario named as "baseline".
+
+        Args:
+            io (str, optional): The path to the add sectors master file containing the inventories. If 'inventories' is passed (default), 
+                                the inventories will be read from the 'inventories' attribute. 
+            scenario (str, optional): The scenario to add the inventories to. Defaults to 'baseline'.
+            inplace (bool, optional): If True, the changes will be applied to the current instance. If False, a new instance will be returned. Defaults to False.
+
+        Raises:
+            ValueError: If the source is not one of the acceptable inventory sources.
+            AttributeError: If the inventories have not been parsed yet. Use read_inventories() first.
+            NotImplementedError: If the source is 'FIONA' (not implemented yet).
+
+        Returns:
+            mario.Database: A new mario.Database instance with the added sectors.
+            
         """
+
         if not inplace:
             new = self.copy()
             new.add_sectors(
-                io=io, new_sectors=new_sectors, regions=regions, item=item, inplace=True
+                io=io,
+                scenario=scenario,
+                inplace=True,
+                ignore_warnings=ignore_warnings,
+                notes=notes,
             )
-            return new
 
-        difference = set(regions).difference(self.get_index(_MASTER_INDEX["r"]))
-
-        if difference:
-            raise WrongInput(
-                "Regions: {} do not exist in the database. Existing regions are:\n{}".format(
-                    difference,
-                    self.get_index(_MASTER_INDEX["r"]),
-                )
-            )
+            return new
 
-        if self.meta.table == "SUT":
-            if item not in [_MASTER_INDEX["c"], _MASTER_INDEX["a"]]:
-                raise WrongInput(
-                    "For SUT, item should be {} or {}".format(
-                        _MASTER_INDEX["c"], _MASTER_INDEX["a"]
-                    )
-                )
         else:
-            item = _MASTER_INDEX["s"]
-
-        log_time(
-            logger,
-            "Database: All the scenarios will be deleted from the database",
-            "warning",
-        )
-
-        new_data, units = add_new_sector(self, io, new_sectors, item, regions)
+            err_msg = []
+            item_to_query = _MASTER_INDEX['a'] if self.meta.table == 'SUT' else _MASTER_INDEX['s']
+            for inventory in self.inventories:
```

Comment:
do we need to have invenotories as a property? better to avoid it or if needed for your code, better to delete it after use